_includes/favicon.html

```diff
@@ -0,0 +1,20 @@
+<link rel="apple-touch-icon" sizes="57x57" href="/images/icons/apple-touch-icon-57x57.png">
```

Comment:
we can probably remove `favicon.html` and the matching include for a single favicon.ico